sdk/servicebus/service-bus/src/serializers/topicResourceSerializer.ts

```diff
@@ -243,6 +256,11 @@ export interface TopicProperties {
    */
   maxSizeInMegabytes: number;
 
+  /**
+   * The maximum message size in kilobytes for messages sent to this queue/topic.
+   */
```

Comment:
Can we add the note about being configurable only for Premium and service version 2021-05 or higher for each property?

Reply:
Added that note for the `Create{Entity}Options` interfaces.
Did not add the same for `{Entity}Properties` interface as that is used for output responses.

Reply:
The users can't configure the service version?

Reply:
It seems like it should be configurable based on the TS guidelines - https://azure.github.io/azure-sdk/typescript_design.html#ts-service-versions-select-api-version

Reply:
Good point.

Should I add this in a future PR, @ramya-rao-a?

Reply:
You can create an issue for it @HarshaNalluru

Reply:
#17972

Reply:
Wouldn't we need to do this before releasing a new service version?

Reply:
@ramya-rao-a

Reply:
Is there a scenario where the customer would want to intentionally use the older API version? If so, then we can make sure we allow that before GA

Reply:
Yes, we recommend that libraries that use the SDK pin to a specific API version. So we should probably support that before GA.